app/mailers/requests_confirmation_mailer.rb

```diff
@@ -15,25 +15,4 @@ def confirmation_email(request)
     cc.uniq!
     mail(to: requester.email, cc: cc, subject: "#{@organization.name} - Requests Confirmation")
   end
-
-  private
-
-  # TODO: remove the need to de-duplicate items in the request
```

Comment:
Going forward (for a while now), duplication isn't allowed when the request is created in the first place, so we don't need to merge things for the email anymore.